docs/samples/explanation/alibi/imagenet/test_imagenet.py

```diff
@@ -0,0 +1,81 @@
+import argparse
+import matplotlib.pyplot as plt
+from tensorflow.keras.applications.inception_v3 import InceptionV3, preprocess_input, decode_predictions
+from alibi.datasets import fetch_imagenet
+import numpy as np
+import requests
+import json
+import os
+from PIL import Image
+
+PREDICT_TEMPLATE = 'http://{0}/v1/models/imagenet:predict'
+EXPLAIN_TEMPLATE = 'http://{0}/v1/models/imagenet:explain'
+
+def get_image_data():
+    data = []
+    image_shape = (299, 299, 3)
+    target_size = image_shape[:2]
+    image = Image.open("cat-raw.jpg").convert('RGB')
+    image = np.expand_dims(image.resize(target_size), axis=0)
+    data.append(image)
+    data = np.concatenate(data, axis=0)
+    return data
+
+def predict(cluster_ip):
+    data = get_image_data()
+    images = preprocess_input(data)
+
+    payload = {
+    "instances": [images[0].tolist()]
+    }
+
+    # sending post request to TensorFlow Serving server
+    headers = {'Host':'imagenet-predict.default.svc.cluster.local'}
```

Comment:
just a note that this will stop working in knative serving 0.9, you are no longer supposed to use internal host name from outside because some service may not get exposed on public ingress gateway.

Reply:
ok will change

Reply:
@yuzisun what's the plan for moving this and testing on KNative v0.1?

Reply:
I have changed to use `example.com`

Reply:
Thanks! I found this yesterday while certificating knative 0.9, the internal host name only works with local gateway and no longer exposed on public ingress way, checked with knative team it was a mistake that it gets exposed on public ingress way in previous releases. We will do a separate PR to address all the docs.